.github/workflows/build_sparkr_window.yml

```diff
@@ -16,7 +16,7 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-name: "Build / SparkR-only (master, 4.4.0, windows-2019)"
+name: "Build / SparkR-only (master, 4.4.0, windows-2022)"
```

Comment:
It actually fails some tests so I used 2019 :-). It would need more fixes

Reply:
Okay,  Let me investigate it carefully. 😄

Reply:
However, the test based on `windows server 2022` surprisingly succeeded, which is amazing.
https://github.com/apache/spark/pull/47021
https://github.com/panbingkun/spark/actions/runs/9655626675/job/26631829756
<img width="616" alt="image" src="https://github.com/apache/spark/assets/15246973/eba26ee7-1fd9-4ee1-84e7-2c1aa1c2fab5">

Reply:
good good

Reply:
Haha, let's continue to observe the scheduling of `build_sparkr_window.yml` every day. ❤️